src/main/java/nl/tudelft/asteroids/game/Difficulty.java

```diff
@@ -0,0 +1,24 @@
+package nl.tudelft.asteroids.game;
+
+/**
+ * Created by Jasper on 10/20/2016.
+ */
+public enum Difficulty {
+
+    EASY(1), MEDIUM(2), HARD(3);
+
+    private int difficulty;
+
+    Difficulty(int difficulty) {
+        this.difficulty = difficulty;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString().charAt(0) + super.toString().substring(1).toLowerCase();
+    }
+
+    public int getDifficulty() {
+        return this.difficulty;
```

Comment:
I've tested the levels and they do seem pretty appropriate, but I agree that hard could get too difficult at higher scores and maybe easy stays too easy at higher scores.